pimd/pim_cmd.c

```diff
@@ -3760,7 +2469,6 @@ static void clear_interfaces(struct pim_instance *pim)
 		return CMD_WARNING_CONFIG_FAILED;			\
 	}
 
-#if PIM_IPV != 6
```

Comment:
Don't touch the MSDP changes for now, we don't know it needs to be implemented or not, we are not going to implement it so lets just remove the commits related to MSDP, keep it in some branch if required later we can raise it from there.

Reply:
This was cherry-picked from your changes, i didn't change anything